src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.cs

```diff
@@ -325,28 +325,6 @@ public static bool HasMonoAndroidReference (MetadataReader reader)
 			return false;
 		}
 
-		public static bool HasResourceDesignerAssemblyReference (ITaskItem assembly)
```

Comment:
I would have left these in the MonoAndroidHelper since they might be useful in other parts of the build system at some point.But I guess we could move them back if they are really needed 🤷 😄